src/main/java/cz/vutbr/web/csskit/OutputUtil.java

```diff
@@ -69,6 +69,31 @@ public static StringBuilder appendTimes(StringBuilder sb, String append, int tim
 	}
 	
 	/**
+	 * Appends all elements of <code>array</code> to buffer, separated by delimiter
+	 * @param <T> Type of elements stored in <code>array</code>
+	 * @param sb StringBuilder to be modified
+	 * @param array Array of elements
+	 * @param delimiter Delimiter to separate elements
+	 * @return Modified <code>sb</code> to allow chaining
+	 */
+	public static <T> StringBuilder appendArray(StringBuilder sb, T[] array, String delimiter) {
+		
+		boolean firstRun = true;
```

Comment:
I haven't tested this yet but I have just a quick question: appendArray() will be probably called repeatedly (e.g. multiple rule sets). Will the firstRun flag work correctly for all repeated calls? Aren't we going to obtain extra delimiters?